sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1172,11 +1173,11 @@ public String apply(TableRow value) {
         }
     );
 
-    SerializableFunction<BoundedWindow, String> tableFunction =
-        new SerializableFunction<BoundedWindow, String>() {
+    TableWritePolicy<String> tableFunction = new TableWritePolicy<String>() {
```

Comment:
Suppose there should also be a RunnableOnService test that actually uses the element.